king_phisher/client/widget/managers.py

```diff
@@ -373,3 +374,61 @@ def signal_activate_popup_menu_copy(self, menuitem, column_ids):
 
 	def signal_activate_popup_menu_delete(self, menuitem):
 		self._call_cb_delete()
+
+class TimeSelector(gui_utilities.GladeGObject):
+	dependencies = gui_utilities.GladeDependencies(
```

Comment:
add in description for the class, as this is designed to be used in other parts of the application if we ever need to select time again

Reply:
I'll add the description, but in this case, the class is reserved to only be used with the TimeSelectorButtonManager. This is the code that instantiates the TimeSelector as a GTK.Popover() and gets fed directly into `TimeSelectorButtonManger(button=TimeSelector, application=self.application))`